### PR TITLE
Update to 8.8.5

### PR DESCRIPTION
**Pull request by Danealau**
---

- update to 8.8.5
- update to 143 layer
- add field "last_layer" to json (last layer for TLObject with id)

### CODE REVIEW

**Review by Rolariali — COMMENTED** [review]
Я бы конечно это не добавлял, если не используем
`    public const PARTICIPANTS_BOTS = 2966521435;

    public const PARTICIPANTS_KICKED = 2746567045;
    public const PARTICIPANTS_BANNED = 338142689;
    public const PARTICIPANTS_SEARCH = 106343499;
    public const PARTICIPANTS_CONTACTS = 3144345741;

    public const PARTICIPANTS_MENTIONS = 3763035371;`

**Review by Rolariali — APPROVED** [review]

File: src/TLMessage/TLMessage/ClientMessages/register_device_push.php [hunk 1]
```diff
@@ -8,38 +8,56 @@
...
+     *
+     * @return string
+     */
+    public function toBinary(): string
+    {
+        return
+                Packer::packConstructor(self::CONSTRUCTOR).
+                Packer::packInt(0). // flags , no_muted=false
+                Packer::packInt(2). // token_type, 2 - FCM (firebase token for google firebase)
+                Packer::packString('eJw6Qq8aGB8:APA91bGmp2tE0mGtW2zs-zk-rOoOAV_ggVeDCAojNuUd3A4Elf47vJKUSWClmJEAb6xae2wuIGZzVSwLvptEmQ9lPZIevuTf_Jbo3RMdXXq_Tebcfh3r9ioVhYRqkGH-sygjXj9e6YES'). // token
```
Comment (Ririconj) [inline src/TLMessage/TLMessage/ClientMessages/register_device_push.php:56]: Bad idea to hardcode it!

**Review by Ririconj — CHANGES_REQUESTED** [review]

File: src/TLMessage/TLMessage/ClientMessages/register_device_push.php (hunk 1 above)
Comment (Danealau) [reply src/TLMessage/TLMessage/ClientMessages/register_device_push.php:56]: It was already in register_device_push_java.php

Comment (Ririconj) [reply src/TLMessage/TLMessage/ClientMessages/register_device_push.php:56]: Kick off this class

**Review by Ririconj — CHANGES_REQUESTED** [review]

**Review by Ririconj — APPROVED** [review]

**Pull request merged**